gsheet2notion.py

```diff
@@ -3,92 +3,158 @@
-    df = pd.DataFrame(
-        googlesheet(gsheet_page))
+    df = pd.DataFrame(_read_google_sheet(gsheet_page))
 
-    df.columns = ['Date', 'Name', 'Email', 'OpenLand', 'Telegram', 'Team_to_help', 'Competence', 'Task', 'Profit',
-                  'Why_we', 'work_hour', 'timezone', 'About_me', 'otkuda_uznal', 'CV', 'first_status',
-                  'Interview', 'Result']
+    df.columns = _COLUMNS
 
     # Convert DataFrame to strings
     df = df.astype(str)
 
-    # Convert to a list of dictionaries
+    # Convert into a list of dictionaries
     df = pd.DataFrame.to_dict(df, orient='records')
 
     # Notion, Access a database using the URL of the database page or the inline block
+    client = NotionClient(token_v2=notion_key)
     cv = client.get_collection_view(notion_url)
 
-    # Read csv
+    # Read the CSV file
     with open(db_file) as f:
-        db = [{k: str(v) for k, v in row.items()}
-             for row in csv.DictReader(f, skipinitialspace=True)]
+        db = [
+            {
+                k: str(v) for k, v in row.items()
+            }
+            for row in csv.DictReader(f, skipinitialspace=True)
+          ]
 
     time_check = []
     for i in db:
         time_check.append(i['Date'])
 
-    # We compare the DataFrame with the file using the 'Date' key and write in 'result'
-    # and add new pair in dictionary
+    # Compare the `DataFrame` with records from the file using the `Date`
+    # key, write to the `result`, and add a new pair to the dictionary.
     result = []
     for i in df:
         if i['Date'] not in time_check:
             result.append(i)
             i.update({'Status': 'Новая'})
 
-    # Write in Notion from result
+    # Write to Notion from the `result`
     for mapping in result:
         row = cv.collection.add_row()
         for key, value in mapping.items():
             setattr(row, key, value)
 
-    # Append in csv file
+    # Append to the CSV file
     try:
         keys = result[0].keys()
         with open(db_file, 'a+', newline='') as output_file:
             dict_writer = csv.DictWriter(output_file, keys)
+            # TODO Write the header to make the result file a bit more
+            # human-friendly but skip the first record on read.
             # dict_writer.writeheader()
             dict_writer.writerows(result)
 
-        # Collecting names for message in Slack
-        namelist = []
-        for el in result:
-            name = el.get('Name')
-            namelist.append(name)
+        # Collecting names and send message to Slack
+        # TODO Move out of `try`
+        msg = ', '.join(map(lambda item: item.get('Name'), result))
+        _send_message_to_slack(
+            slack_channel
+          , text=f'New volunteer CV: {msg}'
+          )
 
-        # Send message in Slack
-        msg = ', '.join(namelist)
-        message(slack_channel, text=f"New volunteer CV: {msg}")
-    except:
-        pass
+    except Exception:
```

Comment:
Exception handling is a crucial part of writing programs %) -- need to learn how to make it right %)